lib/api/controllers/adminController.js

```diff
@@ -46,49 +41,24 @@ class AdminController extends BaseController {
       'loadSecurities',
       'resetCache',
       'resetDatabase',
-      'resetKuzzleData',
       'resetSecurity',
       'shutdown'
     ]);
   }
 
-
-  /**
-   * Reset the internal storage components (internal index, cache and public storage)
-   */
-  resetKuzzleData (request) {
```

Comment:
Remove this method since it can lead to inconsistency with the JWT seed in a cluster environment